app/models/proposal.rb

```diff
@@ -30,12 +30,12 @@ class Proposal < ActiveRecord::Base
   has_many :individual_approvals, ->{ individual }, class_name: 'Approvals::Individual'
   has_many :approvers, through: :individual_approvals, source: :user
   has_many :api_tokens, through: :individual_approvals
-  has_many :attachments
+  has_many :attachments, dependent: :destroy
```

Comment:
Are these `dependent: destroy` attributes intended to implement cascading deletes? I had assumed they were added because of the `paranoia` gem, but since that is no longer part of this PR, are they here as a best practice?